src/java.base/share/classes/sun/net/www/protocol/http/DigestAuthentication.java

```diff
@@ -530,7 +653,8 @@ private String computeDigest(
         "00000000", "0000000", "000000", "00000", "0000", "000", "00", "0"
     };
 
-    private String encode(String src, char[] passwd, MessageDigest md) {
+    // truncate256 means only use the last 256 bits of the digest (32 bytes)
+    private String encode(String src, char[] passwd, MessageDigest md, boolean truncate256) {
         md.update(src.getBytes(ISO_8859_1.INSTANCE));
```

Comment:
Maybe we can support the "charset" parameter as well. The only allowed value is "UTF-8".

Reply:
I'll look into supporting that.